t/schema.t

```diff
@@ -0,0 +1,39 @@
+use strict;
+use warnings FATAL => 'utf8';
+use utf8;
+use open ':std', ':encoding(UTF-8)'; # force stdin, stdout, stderr into utf8
+
+use Data::UUID;
+use Path::Tiny;
+use Test::Conch;
+use Test::Deep;
+use Test::Memory::Cycle;
+use Test::More;
+use Test::Warnings ':all';
```

Comment:
No import is needed for this file, as this is only being loaded for the 'had no warnings' hook in `done_testing`.

Reply:
Should be fixed in bdf6f8d